nomenclature/testing.py

```diff
@@ -37,9 +37,9 @@ def assert_valid_structure(
     path : Path
         directory path to the file of interest
     mappings : str
-        Optionnal non-default name for the mappings folder
+        Optional non-default name for the mappings folder
```

Comment:
Suggest to add a note here that the default folder name is "mappings" because this is not clear from the signature.

Reply:
Done.